### ISSUE TITLE
Add Field for Stroke Order, Character Count, and/or Complexity Index

### ISSUE THREAD

**Issue by Nick3C** [batterseapower/pinyin-toolkit#101]
I think support for a field representing aproximate complexity would be helpful. It would allow sorting in the editor by rough complexity (something I often find myself wanting to do so that I can prioritise learning to simple, unlearned characerts). I would also be \* extremely\* helpful for speaking cards (so you know how many characters you need)

I thought that this could be produced using two methods: length of character field, total strokes.

I see several options:
1) total stroke order (i.e. add all characters strokes together)
2) composite figure eg: "3-25" meaning 3 characters 25 strokes
3) simple arithmetic index (provide an index of complexity using formulae like x=[multiplier]*[no.chars]+ [strokes]
4) hybrid in the format of "[index](3-25)"

I think I like either option 2 or option 4.
I think option 2 might be safer (although option 4 will be neater).

Need to double check how anki would sort each of these first.

I think one of your other plugins already does stroke counts doesn't it?

**Comment by batterseapower** [comment]
Yes, stroke count data is available in Unihan. The need for a proper DB backend with all of unihan, CEDICT CFDICT Handedict etc loaded into it is pressing.
